apps/ttd-app/src/App.tsx

```
@@ -0,0 +1,82 @@
+      </div>
+    );
+  }
+
+  if (state === "error") {
+    return (
+      <div className="error-screen">
+        <div className="error-content">
+          <h1>Failed to Load</h1>
+          <p>{error}</p>
+          <button className="btn btn-primary" onClick={() => window.location.reload()}>
+            Retry
```

Biome (2.3.14)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)